src/gsi/get_gefs_for_regional.f90

```diff
@@ -365,36 +486,53 @@ subroutine get_gefs_for_regional
   iadate_gfs(1)=jda(1) ! year
   iadate_gfs(2)=jda(2) ! mon
   iadate_gfs(3)=jda(3) ! day
+  write(6,*) 'iadate_gfs year=', iadate_gfs(1)
+  write(6,*) 'iadate_gfs mon=', iadate_gfs(2)
+  write(6,*) 'iadate_gfs day=', iadate_gfs(3)
   if(ntlevs_ens > 1) then
      iadate_gfs(4)=jda(5)+hrdifsig(ntguessig)-hrdifsig(it) ! hour
   else
      iadate_gfs(4)=jda(5) ! hour
   endif
   iadate_gfs(5)=0      ! minute
+
+  write(6,*) 'iadate_gfs hou=', iadate_gfs(4)
+
+  write(6,*) 'iadate=', iadate
+  write(6,*) 'iadate_gfs=', iadate_gfs
+
   if(mype == 0) then
      write(6,*)' in get_gefs_for_regional, iadate_gefs=',iadate_gfs
      write(6,*)' in get_gefs_for_regional, iadate    =',iadate
   end if
+
            call w3fs21(iadate,nming1)
            call w3fs21(iadate_gfs,nming2)
+  write(6,*) 'nming1=', nming1
+  write(6,*) 'nming2=', nming2
+  write(6,*) 'l_ens_in_diff_time=', l_ens_in_diff_time
```

Comment:
Same comment as before, the print statements on the previous 20 or so lines should be wrapped like how the existing one is "if(mype ==0)"

Reply:
Removed.